### ISSUE TITLE
Magic Fields data not being saved

### ISSUE THREAD

**Issue by vpjm** [hunk/Magic-Fields#189]
I was running Magic Fields 1.6.3.2 on WordPress 4.5.3 without any issues (that I noticed). 

I updated to WordPress 4.6 and the same version of Magic Fields (1.6.3.2) starting having an issue with saving data in any of the fields. It seems like nothing is being saved. I add data into fields and when I update the page, nothing is saved (the fields are empty when the page reloads). It also returns empty fields if I update a page with previously saved data. 

I tried updating to Magic Fields 1.7.2 still on WordPress 4.6 and the same issue was happening. I then tried going back to WordPress 4.5.3 with Magic Fields 1.6.3.2 (the original setup) and the same problem was happening here. I then tried WordPress 4.5.3 with Magic Fields 1.7.2 with the same problem.

Any ideas would be appreciated.

Thanks!

**Comment by vpjm** [comment]
So it seems like data is being saved to the database but when the page is loaded, the data is not being loaded into the fields.

**Comment by hunk** [comment]
Hi, do you have more plugins?  can you enable debug mode and errors in PHP maybe show some error. already check my version of MF 1.x and works fine.

**Comment by NnattBV** [comment]
It happens exactly the same. I have not found a solution . My site is publish and with serious problems, any solution? Tnk

**Comment by Jasadanor** [comment]
Hi, 

I had the same problem with my site, but it looks like it must be a problem with a plugin that was updated and created a conflict. Not only were fields being emptied, some were being populated with data from other fields. And a date picker would only allow dates from the 1960's. 

My WP version is 4.6.1 with Magic Fields 1.6.3.2, but after I used a backup to revert a ton of plugins that had just been updated, everything was working again. I have not determined which one was the culprit, but if I find out I will post here. And maybe others with the same issue will see a plugin in common to help narrow the list down (sorry for the long list).

Add From Server
You have version 3.3.1 installed. Update to 3.3.2. View version 3.3.2 details.
Compatibility with WordPress 4.6.1: Unknown

Akismet
You have version 3.1.11 installed. Update to 3.2. View version 3.2 details.
Compatibility with WordPress 4.6.1: 100% (according to its author)
	
Category Posts Widget
You have version 4.6.1 installed. Update to 4.6.2. View version 4.6.2 details.
Compatibility with WordPress 4.6.1: 100% (according to its author)

Contact Form 7
You have version 4.4.2 installed. Update to 4.5. View version 4.5 details.
Compatibility with WordPress 4.6.1: 100% (according to its author)
	
Custom Sidebars
You have version 2.1.0.8 installed. Update to 2.1.1.0. View version 2.1.1.0 details.
Compatibility with WordPress 4.6.1: 100% (according to its author)

Jetpack by WordPress.com
You have version 4.1.1 installed. Update to 4.3.1. View version 4.3.1 details.
Compatibility with WordPress 4.6.1: 100% (according to its author)
	
Max Mega Menu
You have version 2.2.2 installed. Update to 2.2.3.1. View version 2.2.3.1 details.
Compatibility with WordPress 4.6.1: 100% (according to its author)
	
Max Mega Menu - Pro Addon
You have version 1.4.5 installed. Update to 1.5.1. View version 1.5.1 details.
Compatibility with WordPress 4.6.1: Unknown
	
Popup with fancybox
You have version 2.2 installed. Update to 2.3. View version 2.3 details.
Compatibility with WordPress 4.6.1: 100% (according to its author)
Tested up to 4.6

SearchWP
You have version 2.8.2 installed. Update to 2.8.4. View version 2.8.4 details.
Compatibility with WordPress 4.6.1: Unknown

SearchWP Term Highlight
You have version 2.0.12 installed. Update to 2.0.13. View version 2.0.13 details.
Compatibility with WordPress 4.6.1: Unknown

TinyMCE Advanced
You have version 4.3.10.1 installed. Update to 4.4.1. View version 4.4.1 details.
Compatibility with WordPress 4.6.1: 100% (according to its author)
	
UpdraftPlus - Backup/Restore
You have version 1.12.17 installed. Update to 1.12.20. View version 1.12.20 details.
Compatibility with WordPress 4.6.1: 100% (according to its author)

Username Changer
You have version 2.0.5 installed. Update to 2.1.1. View version 2.1.1 details.
Compatibility with WordPress 4.6.1: Unknown

WordPress Importer
You have version 0.6.1 installed. Update to 0.6.3. View version 0.6.3 details.
Compatibility with WordPress 4.6.1: 100% (according to its author)

WP Job Manager
You have version 1.25.0 installed. Update to 1.25.1. View version 1.25.1 details.
Compatibility with WordPress 4.6.1: 100% (according to its author)
This release includes many bugs and fixes including some security improvements. Upgrade today!
	
Yoast SEO
You have version 3.4 installed. Update to 3.5. View version 3.5 details.
Compatibility with WordPress 4.6.1: 100% (according to its author)

**Comment by Jasadanor** [comment]
Dan, I have not had time to go through and update each plugin one by one yet, but I will report back as soon as I find out anything.

**Comment by danken00** [comment]
@Jasadanor Thanks pal. I'm just going through mine as well (we share a few plugins in common). I'll let you know if I find anything too.

**Comment by Jasadanor** [comment]
After updating each plugin one by one, it looks like Jetpack 4.1.1 was the one that caused problems. I tried going through each module in Jetpack, but was unable to deactivate them. These are the modules enabled by the plugin. Hope this helps

Custom CSS
Custom Content Types
Enhanced Distribution
Extra Sidebar Widgets
Gravatar Hovercards
JSON API
Protect
Publicize
Sharing
Shortcode Embeds
Site Stats
Spelling and Grammar
Subscriptions
WP.me Shortlinks
Widget Visibility

**Comment by hunk** [comment]
@NnattBV @vpjm @danken00 also you have enabled JectPack?

**Comment by danken00** [comment]
@hunk Yes, I have JetPack installed and activated on my installation. Disabling it seemed to resolve this issue.

**Comment by danken00** [comment]
@Jasadanor Champion, great work. Thanks mate.
